### PR TITLE
VS-1119 remove unused call

### PR DESCRIPTION
**Pull request by gbggrant**
Passing Integration test (using built-in dockers) [here](https://job-manager.dsde-prod.broadinstitute.org/jobs/d2063e26-a22e-412e-ad91-aa2b14fbb7ec).

### CODE REVIEW

**Review by mcovarr — COMMENTED** [review]
tbh I never really knew what this message was trying to tell us so I'm not sure if it's okay to get rid of it or not

**Comment by gbggrant** [comment]
Yeah, I don't know either. But if nobody understands it and it has no effect maybe remove it? I'll discuss it with the team before merging.

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java [hunk 1]
```diff
@@ -244,11 +243,6 @@ public void onTraversalStart() {
         if (outputType == CommonCode.OutputType.BQ) {
             loadStatus = new LoadStatus(projectID, datasetName, loadStatusTableName);
 
-            long streamingBufferRows = BigQueryUtils.getEstimatedRowsInStreamingBuffer(projectID, datasetName, loadStatusTableName);
```
Comment (gbggrant) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java:247]: HI @kcibul  - we've had a little bit of a discussion about this. We're not quite sure what you were looking for here. Is it worth keeping this logging (and underlying method) around?

**Review by RoriCremer — APPROVED** [review]
lets double check with Google about the WriteAPI though

**Pull request merged**